.github/workflows/check.yml

```diff
@@ -496,6 +496,7 @@ jobs:
         if: ${{ steps.app-cache.outputs.cache-hit != 'true' && !contains(github.event.pull_request.labels.*.name, 'ci/skip-detox') }}
         env:
           SKIP_BUNDLING: 'true'
```

Comment:
@hawkrives do we fail to have a bundle because skip_bundling is set to true here?

Reply:
Never mind I see that's only set in the workflow and used for build.gradle